README.md

```diff
@@ -15,51 +15,45 @@ COBOL Language Support standardizes the communication between language tooling f
 
-	- Contrasting colors used in displayed code for ease of identifying and distinguishing keywords, variables, and paragraphs.
+![Syntax check](SyntaxCheckv2.gif)
+			
+### Syntax Highlighting
+The extension enables syntax highlighting for COBOL code.
 	
-- **Copybook Support**
+### Syntax Coloring
+Contrasting colors are used in displayed code for ease of identifying and distinguishing keywords, variables, and paragraphs.
 
-	- Semantic analysis for keywords, variables, and paragraphs, across copybooks supported.
-		- Ensures and maintains compatibility of copybooks called in code.
-	- Inbuilt protection against recursive + missing copybooks.
-		- Protects against code errors where copybook may contain looping code which could crash processes.
-		- Error message shows where copybook is missing, preventing issue only being discovered when code is executed.  
-	- Variables and paragraphs defined across copybooks.
-		- Ensures consistency of code, assisting with error diagnostics as issues will not be caused by incorrect variables or paragraphs in code. 
-	- Autocomplete updated to include copybook variables and paragraphs.
-		- Speeds up coding process by intuitively suggesting most likely variables or paragraphs to follow existing code.
-	- Functionality to skip variable levels when called, reducing call time.
-	- Inbuilt search function to identify all references filled with paragraphs and variables from copybooks:
-		- Find all references:
-			- Easily identifies all occurences of variables and paragraphs in the code
-		- Go To Definition:
-			- Right click on any variable or paragraph to reveal a definition of the element	
+A third-party plugin is required to enable syntax coloring. The Che4z basic stack and Code4z pack both contain Bitlang, which fulfils this requirement.
 	
+### Copybook Support
+To use the COBOL Language Support extension while working with copybooks, place the copybook files inside a **"COPYBOOKS"** folder within the current workspace. The extension includes the following copybook support features:
+
+* Semantic analysis for keywords, variables, and paragraphs across copybooks, to ensure and maintain compatibility of copybooks called in code.
+* Inbuilt protection against recursive and missing copybooks. If the copybook is missing or contains looping code, an error displays, preventing issues only being discovered when the code is executed.  
+* Variables and paragraphs are defined across copybooks. This ensures consistency of code, and prevents issues in error diagnostics caused by incorrect variables or paragraphs in code. 
+* Functionality to skip variable levels when called, reducing call time.
+* Find All References and Go To Definition functionalities.
+** **Find All References** identifies all occurrences of variables and paragraphs from copybooks in the code.
```

Comment:
I think u wanted to indent a sub items here - usually that's done by just SPACE and *

Reply:
Oops, sorry, fixed, forgot this was MD and not adoc.